about/contributing_datasets.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: default
+title: Contributing datasets
+nav_order: 1
+parent: About Data Commons
+---
+
+# Life of a Dataset
+
+This tutorial walks through the process of structuring and inserting data into the Data Commons graph.
```

Comment:
Consider adding links to helpful content / background reading here (or in the background section)

Reply:
I have another PR out for background reading docs migration (#63 ) where background docs are moved to the sidebar; would it be repetitive to also have background docs linked in-text?